test/ngAria/ariaSpec.js

```diff
@@ -952,6 +954,34 @@ describe('$aria', function() {
       expect(clickEvents).toEqual(['div(true)', 'li(true)', 'div(true)', 'li(true)']);
     });
 
+    they('should not prevent default keyboard action if an interactive $type element' +
+      'is nested inside ng-click', nodeBlackList, function(elementType) {
+        function createHTML(type) {
+          var html = '<' + type + '>';
+
+          if (type === 'INPUT' || 'TYPE'  === 'A') {
+            return html;
+          }
+
+          return html + '</' + type + '>';
+        }
+
+        compileElement(
+          '<section>' +
+            '<div ng-click="onClick($event)">' + createHTML(elementType) + '</div>' +
+          '</section>');
+
+        var divElement = element.find('div');
+        var interactiveElement = element.find(elementType);
+
+        // Use browserTrigger because it supports event bubbling
+        browserTrigger(interactiveElement, 'keydown', {cancelable: true, bubbles: true, keyCode: 13});
+        browserTrigger(interactiveElement, 'keydown', {cancelable: true, bubbles: true, keyCode: 32});
+
+        expect(clickEvents).toEqual([elementType.toLowerCase() + '(false)',  elementType.toLowerCase() + '(false)']);
```

Comment:
I think the test would be a bit clearer if you checked `clickEvents` after each `browserTrigger`.
Otherwise someone might read this test as "you need to trigger two keydowns to get any events".

Reply:
Good idea, updated